packages/opentelemetry-shim-opentracing/src/shim.ts

```diff
@@ -72,10 +74,15 @@ function getContextWithParent(options: opentracing.SpanOptions) {
  */
 export class SpanContextShim extends opentracing.SpanContext {
   private readonly _spanContext: api.SpanContext;
+  private _correlationContext: api.CorrelationContext;
 
-  constructor(spanContext: api.SpanContext) {
+  constructor(
+    spanContext: api.SpanContext,
+    correlationContext: CorrelationContext
```

Comment:
Name is `SpanContextShim` but it now shims both. Should we think of a new name? I'm sorry for my unfamiliarity with the OpenTracing API, but is the Correlation accessed through the span context in OT?

Reply:
In opentracing there is no Correlation context, so no direct access to it, the baggage (aka correlation context) is part of the span context. I don't think we need to rename it :) 

According to this: https://opentracing.io/docs/overview/tags-logs-baggage/

> The SpanContext carries data across process boundaries. Specifically, it has two major components:
> 
>     An implementation-dependent state to refer to the distinct span within a trace
>         i.e., the implementing Tracer’s definition of spanID and traceID
>     Any Baggage Items
>         These are key:value pairs that cross process-boundaries.
>         These may be useful to have some data available for access throughout the trace.